capstone/src/main/webapp/explore-loader.js

```diff
@@ -61,12 +90,13 @@ function getLabelQueryString () {
   return queryString.slice(0, -1); // Takes out the last comma. 
 }
 
-function sendJoinedAlert(interestedOrJoin) {
-  var alertMessage = 'You have successfully ';
+function sendJoinedAlert(interestedOrJoin, alreadyJoined) {
+  var alertMessage = 'You have ';
+  alertMessage += alreadyJoined ? 'already ' : 'successfully ';
   if (interestedOrJoin == 'interested') {
     alertMessage += 'expressed interest in this club! You can view your interested club list on your profile page.';
   } else {
-    alertMessage += 'joined a club! You can view your club list or leave a club on your profile page. You will now receive notifications for this club.';
+    alertMessage += 'joined this club! You can view your club list or leave a club on your profile page. You will receive notifications for this club.';
```

Comment:
Would you be able to pass in the name of the club here if it's not too difficult? Would be a nice to have to specify the club name here. No worries if not.

Reply:
Just added the club name! Thank you for pointing this out :)